cloudformationTemplates/urlPreviewForAsyncChat/cloudformation.yaml

```diff
@@ -104,6 +125,7 @@ Resources:
   ### DDB for url preview cache ###
   UrlPreviewCacheTable:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Ensure Dynamodb point in time recovery (backup) is enabled